app/javascript/components/time.jsx

```diff
@@ -1,54 +1,37 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { GreySubText } from './typography';
-import { colors, fontSizes, spacing, weights, fonts } from '../lib/theme';
+import { colors, fontSizes, weights, fonts } from '../lib/theme';
 import { Row } from './row';
 import { timeForTimezone } from '../lib/datetime';
 
 const Wrapper = styled.div`
   font-size: ${fontSizes.medium};
-  font-weight: ${weights.light};
 `;
 
-const HourMin = styled.div``;
+const HourMin = styled.div`
+  font-size: ${fontSizes.xlarge};
+  font-family: ${fonts.extended};
+  font-weight: ${weights.regular};
+  position: relative;
+  top: 8px;
+`;
 
-const AMPM = styled.div``;
+const AMPM = styled.div`
```

Comment:
Nit: could be named `Meridiem`. Might just be more confusing but I initially thought `AMPM` was an acronym.